server/app/controllers/fusor/api/v21/deployments_controller.rb

```diff
@@ -40,8 +40,7 @@ def update
       # controller (after it has validated them), never by directly updating
       # the deployment object.
 
-      # commented out :openstack_undercloud_password since mock UI application needs to set this value
-      # params[:deployment].delete :openstack_undercloud_password
+      params[:deployment].delete :openstack_undercloud_password
```

Comment:
@cfchase, @jmontleon, I just made the change this week where I commented out the line. I didn't realize the side effect and still a little confused why/how Ember is causing the value to change to nil. I will look more on Sunday, but it's good that you merged.